src/sciwrappers.h

```diff
@@ -218,6 +219,8 @@ void				sci_move_selected_lines_up      (ScintillaObject *sci);
 
 void				sci_set_font_fractional		(ScintillaObject *sci, gint style, const gchar *font, gdouble size);
 
+gint				sci_get_fontsize			(ScintillaObject *sci, gint style);
```

Comment:
Rename to `sci_get_font_size()` here as well

Reply:
Done.